### PR TITLE
EN-1749 [Python SDK] Improve integration tests

### PR DESCRIPTION
**Pull request by mayya-miftakhova**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- https://incountry.atlassian.net/browse/EN-1749

Code Quality (Before submitting PR)
------------
- [ ] Ensure code compiles and passes Sonar Lint.
- [ ] Do your best to ensure integration tests pass. If they don't then...
- [ ] Remove, ignore, or mark pending any brittle tests.
- [ ] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version? What is the new version?

### CODE REVIEW

File: tests/integration/popapi_integrations_test.py [hunk 1]
```diff
@@ -1,47 +0,0 @@
...
-from incountry import Storage
-from pytest_testrail.plugin import pytestrail
-from incountry import SecretKeyAccessor
-import uuid
-
-
-API_KEY = os.environ.get("INT_INC_API_KEY")
-ENVIRONMENT_ID = os.environ.get("INT_INC_ENVIRONMENT_ID")
-ENDPOINT = os.environ.get("INT_INC_ENDPOINT")
-COUNTRY = os.environ.get("INT_INC_COUNTRY")
```
Comment (RommeroInCountry) [inline tests/integration/popapi_integrations_test.py]: @mayya-miftakhova please note that we've added INT_INC_COUNTRY variable to be able to adjust tests to use specific country code in our requests (e.g. if we want to test against se, not us, due to us being down for a couple of days but we still need those tests to pass)

Comment (mayya-miftakhova) [reply tests/integration/popapi_integrations_test.py]: Changed hardcoded country to environment variable

File: tests/integration/test_methods.py [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+def test_read_record(storage: Storage, encrypt: bool) -> None:
+    record = {
+        "key": "unique key",
+        "key2": "some key2",
+        "key3": "some key3",
+        "profile_key": "some profile_key",
+        "body": "some body",
+        "range_key": 42,
+    }
+    storage.write(country=COUNTRY, **record)
```
Comment (tony-ic) [inline tests/integration/test_methods.py]: Don't see that test delete created record after finishing.

Comment (mayya-miftakhova) [reply tests/integration/test_methods.py]: Added test data deletion using fixture clean_up_records

**Pull request merged**